examples/fuzzing/calculator_test.go

```diff
@@ -1,79 +1,88 @@
 package main
 
-/* Fuzz testing or fuzzing is an automated software testing method
-that injects invalid, malformed, or unexpected inputs into a system
-to reveal software defects and vulnerabilities.
-A fuzzing tool injects these inputs into the system and then monitors for exceptions such as crashes or information leakage
-*/
-
-//A simple way of using fuzzing to test a program using a claculator instance
 import (
+	"bytes"
 	"errors"
-	"fmt"
+	"math/rand"
+	"strconv"
 	"testing"
 )
 
 /*A calculator for performing simple arithmetic operations*/
-func calculator(num1, num2 float64, operator string) (float64, error) {
+
+/*
+[]byte was used as its parameters in the Calculator() cause fuzz test  requires []bytes as
+ it parameters when testing a function
+ then the []bytes are coverted to []float for performing arithmetic operations
+*/
+
+func Calculator(numbers []byte) (float64, error) {
+	bytesList := bytes.Split(numbers, []byte(","))
+
+	//convert to float
+	var floatList []float64
+	for _, b := range bytesList {
+		f, err := strconv.ParseFloat(string(b), 64)
+		if err != nil {
+			return 0, err
+		}
+		floatList = append(floatList, f)
+	}
+
 	var result float64
+	var operator string
 
+	//performing calculations based on the indices of the values in FloatList
 	switch operator {
 	case "-":
-		result = num1 - num2
+		if floatList[0] > floatList[1] || floatList[0] < floatList[1] {
+			result = floatList[0] - floatList[1]
+			return result, nil
+		} else if floatList[1] > floatList[0] || floatList[1] < floatList[0] {
+			result = floatList[1] - floatList[0]
+			return result, nil
+		}
 
 	case "+":
-		result = num1 + num2
+		if floatList[0] > floatList[1] || floatList[0] < floatList[1] {
+			result = floatList[0] + floatList[1]
+			return result, nil
+		} else if floatList[1] > floatList[0] || floatList[1] < floatList[0] {
+			result = floatList[1] + floatList[0]
+			return result, nil
+		}
 
 	case "/":
-		if num2 == 0 || num1 == 0 {
+		if floatList[0] == 0 || floatList[1] == 0 {
 			return 0, errors.New("Division by 0")
 		}
+		result = floatList[0] / floatList[1]
+
 	case "*":
-		result = num1 * num2
+		result = floatList[0] * floatList[1]
 	}
 
 	return result, nil
 }
 
```

Comment:
Based on the code i just pushed , i created and utilized the []byte parameter which i converted to a []float to perform the calculations . And i Performed the Calculations based on the indices of the FloatList